lambdas/build-client-oauth-response/src/main/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandler.java

```diff
@@ -85,15 +88,14 @@ public BuildClientOauthResponseHandler(
     @Override
     @Tracing
     @Logging(clearState = true)
-    public APIGatewayProxyResponseEvent handleRequest(
-            APIGatewayProxyRequestEvent input, Context context) {
+    protected JourneyResponse handleRequest(JourneyRequest input, Context context) {
 
         LogHelper.attachComponentIdToLogs();
 
         try {
-            String ipvSessionId = RequestHelper.getIpvSessionIdAllowNull(input);
+            String ipvSessionId = getIpvSessionIdAllowNull(input);
+            String ipAddress = getIpAddress(input);
             String clientSessionId = RequestHelper.getClientOAuthSessionId(input);
```

Comment:
Could make this a static import like the others?

Reply:
done